src/app/beer_garden/systems.py

```diff
@@ -150,7 +136,8 @@ def update_system(
                         f"different commands"
                     )
 
-                system.upsert_commands(new_commands)
+                # TODO - Move this somewhere
```

Comment:
TODO: this.

Reply:
Sigh. I spent 20 minutes trying to figure out why my local plugins didn't have commands and this was the reason. :man_facepalming:

Reply:
Fixed by 407cdb4